src/test/java/net/sf/jabref/importer/ImportFileFilterTest.java

```diff
@@ -0,0 +1,84 @@
+package net.sf.jabref.importer;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import net.sf.jabref.importer.fileformat.ImportFormat;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ImportFileFilterTest {
+
+    @Test
+    public void nameWithSingleExtensions() {
+        ImportFormat importFormatSingleExtension = new ImportFormat() {
+            @Override
```

Comment:
Could you pls use Mockito for mocking the ImportFormat. This is more readable. (Just create a new small PR)